File: storage_sdk/src/main/java/com/walker/storage/room/Word.java
package com.walker.storage.room;

import androidx.annotation.NonNull;
import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

/**
 * Created by walkerzpli on 2021/7/30.
 */

@Entity(tableName = "word_table")
public class Word {

    @PrimaryKey(autoGenerate = true)
    private int id;

    @NonNull
    @ColumnInfo(name = "content")
    private String mContent;


    public Word(@NonNull String content) {
        this.mContent = content;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getId() {
        return id;
    }

    public String getContent() {
        return mContent;
    }

}

File: storage-ktx/src/main/java/com/demo/storage_ktx/Word.kt
package com.demo.storage_ktx

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

/**
 * Created by lizhiping on 2023/6/5.
 * <p>
 * description
 */
@Entity(tableName = "word_table")
data class Word(
    @PrimaryKey val id: Int? = null,
    @ColumnInfo(name = "word") val word: String
)
